### PR TITLE
Webpack integration

### PR DESCRIPTION
**Pull request by gcollazo**
Este PR es para completar #84 reemplazando Brunch por Webpack

### CODE REVIEW

**Comment by gcollazo** [comment]
Test error looks unrelated to this PR

File: README.md [hunk 1]
```diff
@@ -91,18 +91,29 @@ Abre tu browser en [http://localhost:8000/](http://localhost:8000/). Para accesa
 
 
 #### Opción 2: Local
-Para esta opción debes tener instalado **Python 2.7** en tu máquina y [Brunch](http://brunch.io). También es recomendado que crees un [virtualevn](http://www.virtualenv.org/) para el proyecto pero no es un requisito.
+Para esta opción debes tener instalado **Python 2.7** en tu máquina y [Node.js](https://nodejs.org). También es recomendado que crees un [virtualevn](http://www.virtualenv.org/) para el proyecto pero no es un requisito.
```
Comment (froi) [inline README.md]: Add something like:

```
Optional: Install Yarn: <url>
```

**Comment by froi** [comment]
I'll fix the test in another branch so you can rebase.

**Comment by gcollazo** [comment]
@froi creo que este ultimo commit responde a tus comments

**Comment by froi** [comment]
@gcollazo Fixed the test rebase with master and it should work.

**Comment by gcollazo** [comment]
@froi rebased

**Pull request merged**